src/embed.js

```diff
@@ -1,4 +1,4 @@
-import * as prettier from "prettier";
+import * as doc from "prettier/doc";
```

Comment:
Not sure you still want import from the `doc` entry, but this has been fixed in [`3.0.0-alpha.3`](https://github.com/prettier/prettier/releases/tag/3.0.0-alpha.3)